nifi-extension-bundles/nifi-standard-services/nifi-oauth2-provider-api/pom.xml

```diff
@@ -22,4 +22,10 @@
     </parent>
     <artifactId>nifi-oauth2-provider-api</artifactId>
     <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
```

Comment:
Introducing this dependency at the API level seems problematic. At minimum, this should be limited to the jackson-annotation library. However, I recommend an alternative approach that handles the custom fields through an ObjectMapper configuration in the implementation class.